src/Tizen.NUI/src/public/GLWindow.cs

```diff
@@ -392,13 +401,18 @@ public GLWindow.GLWindowOrientation GetCurrentOrientation()
         [EditorBrowsable(EditorBrowsableState.Never)]
         public void SetAvailableOrientations(List<GLWindow.GLWindowOrientation> orientations)
         {
+            if (orientations == null)
+                throw new ArgumentNullException(nameof(orientations));  // To fix CA1062 issue.
```

Comment:
This is not related to this commit(Fix CA2000 warning issues of GLWindow / ViewInternal.`).
Please separate this into different commit or PR for atomic commits.

Reply:
Done
About warning CA1062, remove it from current patch.